en/pragma.tex

```diff
@@ -304,6 +304,20 @@ \subsubsection{} In casual conversation the reflexive perfective of
 \NTeri{3/8/2011}{http://naviteri.org/2011/08/new-vocabulary-clothing/}
 
 
+\subsubsection{Slang} In informal contexts some constructions that
+deviate from standard grammar may be used to serve certain purposes.
```

Comment:
This sentence, while accurate, is not very informative. Can we come up with something more specific? Maybe something like:

> At the request of the community, Paul Frommer approved some constructions deviating from standard grammar that could be used as `slang' among the Na'vi in informal contexts, or as a way to have fun with the language.